
# Vehicle frame

Header header

geometry_msgs/Point[] position

int32   line_type
float64 line_width
